### PR TITLE
2101 UG: Centralise logo

### PR DESCRIPTION
**Pull request by SelwynAng**
Replaced markdown format for the CCACommander logo with HTML tags to introduce styles for centralising the logo.

### CODE REVIEW

**Review by andrechuakj — APPROVED** [review]
LGTM

**Review by wr1159 — COMMENTED** [review]
Should we add this styling in the css so it's applicable for all images?

**Comment by SelwynAng** [comment]
> Should we add this styling in the css so it's applicable for all images?

Sure, I will create a style in `guide.scss` for the 2101 UG to include styling for centralising all images.

File: docs/2101UserGuide.md [hunk 1]
```diff
@@ -312,13 +316,15 @@ Here is an example command where we want to view the events in which the **membe
 
 <figure>
     <img src="images/viewMemberPreCommand.png"
-         alt="viewMember Pre Command">
+         alt="viewMember Pre Command"
+         style="centralise-img">
```
Comment (wr1159) [inline docs/2101UserGuide.md]: Similar with the other instances
```suggestion
         class="centralise-img">
```

**Review by wr1159 — CHANGES_REQUESTED** [review]
I believe we have to change `style` to `class="centralise-img"` and change the `guide.scss` `centralise-img` to `.centralise-img` to make it work for classes.

Alternatively we can add `display: block and margin: 0 auto` in `_base.scss` under img.

Edit: I just realised we are not importing the `guide.scss` into the UG so maybe the second choice is easier,

**Review by wr1159 — APPROVED** [review]
LGTM now, thanks!

**Pull request merged**